korman/ui/ui_menus.py

```diff
@@ -30,6 +30,7 @@ def draw(self, context):
         layout = self.layout
 
         layout.operator("mesh.plasma_ladder_add", text="Ladder", icon="COLLAPSEMENU")
+        layout.operator("mesh.plasma_flare_add", text="Lens Flare", icon="PARTICLES")
```

Comment:
Lamp Flare might be better? I would say a lens flare is a camera effect.

Reply:
Whoops, missed this in the rename.